userspace/libsinsp/test/cri_settings.ut.cpp

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+#include <gtest/gtest.h>
```

Comment:
Not sure if the CI failures are related to that, but I recall there were hiccups including the cri.hpp, e.g. see https://github.com/falcosecurity/libs/blob/ad3e824cdff5e5e68fe7b95d51cd8f64ee9dc915/userspace/libsinsp/test/container_engine/container_parser_cri_containerd.ut.cpp#L19